AGSEC-206/AGSEC-207 - Update the security landing page for ag.org 2.0

abstractj opened the pull request:
Good morning, this PR update our security landing page for something more realistic. I'm still working at the guides, but decided to submit it for immediate feedback.

Updates:
- Removing html and replacing by markdown
- Rephrasing some sentences at the main page
- Organizing sections


@danielpassos @cvasilak @lfryc would you mind to review?

abstractj commented:
I update this pull request to include the sections which I'm working for the guide. Also "In progress" was included and will be removed before we ship it. I'm still working on the other sections.

danielpassos commented:
The `Get started` button seems a little bigger, not sure if it's correct

danielpassos commented:
Not sure if it's related of this PR but the roadmap needs some design improvements

abstractj commented:
@danielpassos the roadmap is based on Jira. Not sure exactly what do you mean, could you please be more specific?

danielpassos commented:
The bigger button ![2015-03-06 07-18-13](https://cloud.githubusercontent.com/assets/13337/6523717/094b0170-c3d1-11e4-9c43-495ec91de40a.png)

danielpassos commented:
About the roadmap, the page layout/design seems strange for me

![screenshot 2015-03-06 07 20 22](https://cloud.githubusercontent.com/assets/13337/6523756/6c1928ae-c3d1-11e4-80e0-67d2f6cbc766.png)

abstractj commented:
@danielpassos regarging the bigger button, this is a patter http://staging-aerogearsite.rhcloud.com/push/.

Regarding the screenshot, that's not the security roadmap and also not part of this change. @matzew created that page a it will be moved.

matzew commented:
> @matzew created that page a it will be moved.

Not sure I understand

abstractj commented:
@matzew OAuth2 page will be moved to the proper section, that's all :)

matzew commented:
ah! :-) cool!

On Fri, Mar 6, 2015 at 12:10 PM, Bruno Oliveira <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> OAuth2 page will be moved to the
> proper section, that's all :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/532#issuecomment-77543252>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

lfryc commented:
@danielpassos the `Getting Started` is correct in terms of consistency with other pages

Nice start, @abstractj ! Not sure which is still in progress, but I'll try:
* let's rename Roadmap button in header OR rather move it to appropriate subsection of the doc (under OAuth2 links?) - it's strange when general button (Roadmap) links to specific section (OAuth2)
* `Guides` button is incorrect as already reported by @danielpassos

cvasilak commented on getstarted/guides/index.markdown:
```diff
@@ -47,20 +47,23 @@ Designed to get you started with enterprise mobile development.
-* [Authorization](/docs/guides/aerogear-ios-2.X/Authorization)
+#### 2. OTP
+* [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
+* [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
```
small link typos:
s/ (AeroGear-OTP) / (/docs/guides/AeroGear-OTP)
s/ (aerogear-cordova/AerogearCordovaOTP) / (/docs/guides/aerogear-cordova/AerogearCordovaOTP)

cvasilak commented on getstarted/guides/index.markdown:
```diff
@@ -47,20 +47,23 @@ Designed to get you started with enterprise mobile development.
+* [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
+* [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
 
-<div class="alert alert-info" role="alert">If you are looking for AeroGear iOS 1.x series, please visit <a href="/docs/guides/aerogear-ios/">this page</a> for more information.</div>
```
feel free to remove this '1.x series note' is already mentioned in up in the page and doesn't add any value

cvasilak commented:
I really like the consolidation of the 'OAuth2 setup for various providers" in the "Before you get started" page, really nice

abstractj commented:
@lfryc updated and rebased after your considerations. I missed the damn buttons mentioned by @danielpassos.

@cvasilak thanks bro, I appreciate it. Updated the PR with your considerations

lfryc commented:
:+1: for merging

@danielpassos @cvasilak can I merge this?

abstractj commented:
Rebased after the latest updates.

danielpassos commented:
@lfryc :+1:

lfryc commented:
Merging...

lfryc commented:
great job @abstractj 

landed :rocket:  on the :moon:
